README.md

```diff
@@ -231,3 +231,218 @@ var_dump($taichungService->getAction1()->do()->getMeaningData());
 var_dump($taichungService->getAction2()->do()->getMeaningData());
 var_dump($taichungService->getAction2()->do()->getMeaningData());
 ```
+
+### HTTP JSON RPC Single Request
+
+In the `Action` object, you can pass the desired `method`, `params`, and `id` for the `RPC` connection by using the `setRpcQuery()` method.
+
+Once the `setRpcQuery()` method is enabled, the HTTP verb of the `Action` will automatically change to `POST`.
+
+By setting the `doneHandler`, you can define the execution logic when the connection is successful, and you can temporarily store the required data in the `Action` instance using `setMeaningData`.
+
+If the `RPC` response is parsed correctly, you can handle your `RPC` response using the `doneHandler`.
+
+You can use `getRpcResponse()` to obtain the successful RPC response entity, `getRpcResult()` to get the RPC response data (i.e., result), and `getId()` to retrieve the RPC response ID. All of these methods return their results as arrays.[Reference to JSON-RPC format](https://www.jsonrpc.org/specification "Reference to JSON-RPC format")。
+
+```php
+require './vendor/autoload.php';
+use SDPMlab\Anser\Service\Action;
+use Psr\Http\Message\ResponseInterface;
+use SDPMlab\Anser\Service\ServiceList;
+
+$id = 1;
+$action = (new Action(
+    'http://myRpcServer.com',
+    "POST",
+    "/"
+))
+->setRpcQuery("/myRpcMethod", [1,2], $id)
+->doneHandler(static function(
+    ResponseInterface $response,
+    Action $runtimeAction
+) {
+    $rpcResponse = $action->getRpcResponse();
+    $result = $action->getRpcResult()[0];
+    $id     = $action->getId()[0];
```

Comment:
I think `$action->getRpcQueryId()` is better than `$action->getId()`.

Reply:
If we use the `batchQuery` pattern, then the return value of `$action->getId()` will become uncertain. I don't think we should provide this method, you should integrate the Query ID with the response structure.

Reply:
Oh~ sorry.
About this, I use getRpcId as the name of the method, the description on README.md is wrong, I will change it in the next commit.

Reply:
So we don't need the `getRpcId()` method, right?
We just need to bind the `rpcid` and `rpcResponse` to the data structure as key-value.

Reply:
Done.